magpie/api/management/user/user_utils.py

```diff
@@ -191,10 +190,10 @@ def get_usr_res_perms():
 
     perm_names = ax.evaluate_call(
         lambda: get_usr_res_perms(),
-        fallback=lambda: db_session.rollback(), httpError=HTTPInternalServerError,
-        msgOnFail=s.UserServicePermissions_GET_NotFoundResponseSchema.description,
+        fallback=lambda: db_session.rollback(), http_error=HTTPInternalServerError,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Lambda may not be necessary](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)